### ISSUE TITLE
question about loss

### ISSUE THREAD

**Issue by icefengzhe** [Tramac/awesome-semantic-segmentation-pytorch#149]
loss震荡比较厉害并且下降一点后就不再收敛，我调整了一些超参依旧没有解决问题，请问关于超参的调试你有什么好的建议？

**Comment by Tramac** [comment]
用的什么数据集，哪个网络呢？

**Comment by lovetian0910** [comment]
我也遇到这个问题，用的Bisenet和自己的数据集，dataloader是仿照cityscapes.py写的。
大佬要是方便的话能否加个微信指导一下，微信号：dukewilliam2010

**Comment by icefengzhe** [comment]
已加你好友，莫浮



发自我的iPhone


------------------ Original ------------------
